internal/handlers/common/error.go

```diff
@@ -149,6 +150,11 @@
 		return writeErr, true
 	}
 
+	var validationErr *wire.ValidationError
+	if errors.As(err, &validationErr) {
+		return NewCommandError(ErrBadValue, err).(*CommandError), true //nolint:errorlint // false positive
```

Comment:
I know that there's a similar line in this function, but why does it exist? Isn't it possible to just return the value from `NewCommandError`?

Reply:
`NewCommandError` return type is `error`, `ProtoErr` interface requires 3 missing methods.